webpack.config.js

```diff
@@ -2,6 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 var lodash = require('lodash');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
+var getModuleDir = require('./tasks/get-module-directory').getModuleDir;
```

Comment:
you can use destructuring here `const {getModuleDir} = require...` (and also in the task file)

Reply:
I got tired of reverting stuff due to incompatible node versions so I just stick to es5 for all node scripts now

Reply:
They work since node 6

Reply:
It might be the case for destructuring, but in general, there are different requirements for different features and I'm too lazy to check it. We probably have 500 LOC of node code, so I don't think it makes a difference either.